### DISCUSSION TITLE
Bypass blocking with a proxy or VPN :

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3866] [Ideas]
Hello, 

 Can you add the functionality of blocking Proxies or VPNs for a host?

 I would like to avoid bypassing the AdGuard Home DNS server with a VPN or a Proxy.

**Comment by ainar-g** [comment]
I'm not sure if that can be done in general case.  You could try and add the domain names of the servers of most VPN providers, but they could still connect with well known IPs.

@ameshkov, do you know of any other way, besides the above?

**Reply by ameshkov** [reply]
I am not entirely sure that this is what's requested.

@admedia85 could you please supply a little bit more details.

**Reply by ghost** [reply]
Hello, 

 In fact, I want to block the use of VPN or Proxies using AdGuard Home. Is this possible?

**Reply by ameshkov** [reply]
Ah, got it. Well, probably there're blocklists that specialize at it.

Try looking for them here: https://filterlists.com/

**Reply by ghost** [reply]
Hello, 

 Think "specialized blocklists" will do?
